frontend/javascripts/admin/voxelytics/statistics_tab.tsx

```diff
@@ -85,12 +90,27 @@ export default function StatisticsTab({
           {statisticsResult.value.map((row: VoxelyticsChunkStatistics) => (
             <tr key={row.executionId}>
               <td>
-                {row.executionId}
-                <br />
-                <span className="stats-label">
-                  {row.countFinished !== row.countTotal && <>{row.countFinished} of </>}
-                  {row.countTotal} chunk{row.countTotal !== 1 && "s"} completed
-                </span>
+                <Tooltip
+                  overlay={
+                    <>
+                      {row.counts.complete}/{row.counts.total - row.counts.skipped} complete,{" "}
+                      {row.counts.cancelled} cancelled, {row.counts.failed} failed,{" "}
+                      {row.counts.skipped} skipped
+                    </>
+                  }
+                >
+                  <span>
+                    {row.executionId}
+                    <br />
+                    <span className="stats-label">
+                      {row.counts.complete !== row.counts.total - row.counts.skipped && (
+                        <>{row.counts.complete} of </>
+                      )}
+                      {row.counts.total - row.counts.skipped} chunk
+                      {row.counts.total - row.counts.skipped !== 1 && "s"} completed
```

Comment:
```suggestion
                      {row.counts.total - row.counts.skipped} {pluralize("chunk", row.counts.total - row.counts.skipped} completed
```

needs to be imported